src/Gui/NaviCube.cpp

```diff
@@ -296,24 +296,21 @@ void NaviCube::setFontSize(int size)
 // the Helvetica font is a good start for most OSes
 QFont NaviCube::getDefaultSansserifFont()
 {
-    QFont font(QString::fromLatin1("Helvetica"));
-    if (font.styleHint() & QFont::SansSerif)
+    // On Windows 10 or newer there is no Helvetica font and Arial will be used
+    // On Win 11 sometimes the first font registered to Windows is used.
+    // Both can lead to ugly, even unreadable results
```

Comment:
These 3 lines of comment describe an older situation and are therefore hard to understand for readers unaware of that situation.

Reply:
No, this is the current situation and the reason why I implemented the setting of the font. On Win 11 (at least my new laptop) the simple check for Helvetica delivers nothing and then the first font in Windows list is used. On Win 10 Arial is used. This is the current situation. This comment should make clear why we first select Bahnschrift for Win, then check for Helvetica.

Reply:
Well, then the text does not make sense to me. 'On Windows 10 or newer' includes Windows 11 obviously. It is confusing that you then go on the mention win 11 separately.

Why not something like:
Windows versions since 2017 have the 'Bahnschrift' font (a condensed sans serif font), we first check for that. If that font is not found we let Qt look for 'Helvetica' or a default sans serif font.

Reply:
Thanks. I updated the comment now accordingly.

However, does the PR work for you?